specification/appconfiguration/AppConfiguration/routes.tsp

```diff
@@ -302,7 +302,7 @@ op getKeyValue is Foundations.ResourceOperation<
     ...ClientRequestIdHeader;
```

Comment:
I am not sure whether this request header should also change, and if so, should we set this to `RequestIdResponseHeader`, or just remove it all together?

The AppConfig Rest API docs don't specify.

Reply:
We have it documented here: https://learn.microsoft.com/en-us/azure/azure-app-configuration/rest-api-headers